Sources/String+Extensions.swift

```diff
@@ -2,25 +2,25 @@ import Foundation
 
 extension String {
 
-  func split(delimiter: String) -> [String] {
-    let components = componentsSeparatedByString(delimiter)
+  func split(_ delimiter: String) -> [String] {
+    let components = self.components(separatedBy: delimiter)
     return components != [""] ? components : []
   }
 
-  func replace(string: String, with withString: String) -> String {
-    return stringByReplacingOccurrencesOfString(string, withString: withString)
+  func replace(_ string: String, with withString: String) -> String {
```

Comment:
@onmyway133 Would you mind to remove this functions an use `replacingOccurrences(of: string, with: withString)` instead?